src/lib/components/event-details.svelte

```diff
@@ -3,25 +3,32 @@
 
   import CodeBlock from './code-block.svelte';
 
+  const shouldDisplay = (key: string, value: unknown): boolean => {
+    if (value === null) return false;
+    if (value === undefined) return false;
+    if (value === '') return false;
+    if (value === '0s') return false;
+    if (key === 'type') return false;
+    return true;
```

Comment:
If we need to expand this it may be useful to check for existence in an object instead of a series of if's totes fine right meow